### ISSUE TITLE
[feature-request] turn off case sensitivity?

### ISSUE THREAD

**Issue by ljodell** [marcel-goldammer/obsidian-keyword-highlighter#28]
### Description

As it is right now, terms are case sensitive. If I have the word "close" as a highlight, "Close" or "CLOSE" do not activate the highlighting.

### Proposed Solution

an option to turn off case sensitivity, or have case sensitivity be off as a default?

### Alternatives Considered

if the possibility to put multiple terms per line were possible, then usual case differences (close, Close, CLOSE) can all be added.

**Comment by Zilsemik** [comment]
I agree.   I'm working on my PhD and using Obsidian. I have dissertation-specific terms in my notes that should not be case sensitive.
